src/reducers/clusterReducer.js

```diff
@@ -92,14 +94,14 @@ export default function clusterReducer(
       });
 
       return {
-        lastUpdated: state.lastUpdated,
+        lastUpdated: Date.now(),
         isFetching: false,
         items: items,
       };
 
     case types.CLUSTERS_LOAD_ERROR:
       return {
-        lastUpdated: Date.now(),
+        lastUpdated: state.lastUpdated,
```

Comment:
This is an unrelated fix. My thinking is that `lastUpdated` should only be set in the success case, not in the error case. It should carry the time when the data has been last fetched successfully.